geniza/corpus/models.py

```diff
@@ -488,6 +488,12 @@ def metadata_prefetch(self):
         # NOTE: footnotes likely should be prefetched depending on use case,
         # but nested prefetching may vary
 
+    def get_by_any_pgpid(self, pgpid):
+        """Find a document by current or old pgpid"""
+        return self.filter(
+            models.Q(id=pgpid) | models.Q(old_pgpids__contains=[pgpid])
+        ).first()
```

Comment:
should this be `get()`? from what I remember, we had validation to prevent duplication of old ids - seems like if there's a duplicate it would be a problem we shouldn't ignore. (although then I suppose you'll have to catch the error when you call it)